src/index.html

```diff
@@ -1,12 +1,45 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
+    <link rel="stylesheet" href="./styles/reset.css">
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
You can combine all the styles in one file and connect only one file, it will increase the performance.

Reply:
Please advise on this one, do you mean totally getting rid of reset.css? Because in the other task your colleague said to keep zeroings in this file.

Reply:
![image](https://github.com/mate-academy/layout_calendar/assets/110335429/c2d3e860-6cf0-495f-90c0-e8205235b903)

Reply:
Meanwhile, I'm submitting a commit implementing the rest of your remarks.

Reply:
Or did you mean that I should import reset.css in main.scss file?.. 

Done!